framework/ui/src/main/java/com/google/android/catalog/framework/ui/CatalogScreen.kt

```diff
@@ -104,37 +118,91 @@ internal fun CatalogScreen(
         }
     }
 
+    val isExpandedScreen = isExpandedScreen()
+    val saver = listSaver<CatalogSample?, String>(
```

Comment:
Nit: It seems like the saved list only has one element, so it probably doesn't need to be a `listSaver` and could probably just be a normal `Saver`?

Reply:
I somehow missed there was a single Saver and though I always had to provide either listSaver or mapSaver. The docs didn't show a single Saver https://developer.android.com/jetpack/compose/state#restore-ui-state